terraform/dashboards/Near/chain-signatures/chain_sig_new.json

```diff
@@ -3028,7 +2936,7 @@
               "sort": "none"
             }
           },
-          "pluginVersion": "11.4.0-77383",
+          "pluginVersion": "11.4.0-79146",
```

Comment:
What exactly is this plugin? We should be careful on updates and make sure there are no breaking changes for plugins.

Reply:
I was only clicking UI, all the changes were made automatically. You can test it locally.